src/login/LoginOnboarding.tsx

```diff
@@ -78,7 +78,9 @@ export default function LoginOnboarding({
         </View>
         <TouchableOpacity
           style={styles.laterButton}
-          onPress={navigation.goBack}
+          onPress={() => {
+            navigation.navigate('ConsequencesFromLoginOnboarding');
```

Comment:
Should this do something special when enable_ticketing is false? 🤔

Reply:
It's the log-in onboarding for periodic tickets. So it's actually not needed.  
`Log in to purchase 7, 30, 60, 90 or 180-day tickets`